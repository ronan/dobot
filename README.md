# Textmark

## About

A simple, text-based autonomation tool.

To get started run: `tasklist new`
