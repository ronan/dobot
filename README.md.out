# Untitled DoBot Project

## About

**Version:** `0.0.1`

## TODO

- [ ] Test Todos

## Changelog

## [0.0.1] - 2023-12-13

- **Added:** README.md file created automatically at ./README.md for **Ronan Dowling**

## Contributors

- **ðŸ¤– DoBot:** <https://github.com/ronan/dobot>
- **Ronan Dowling:** <ronan@dowlings.me>
