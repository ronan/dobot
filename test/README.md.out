# Untitled DoBot Project

## About

**Version:** `0.0.1`

## TODO

- [x] Test Todos

  > Hello, world! I'm testing the todos

  - [ ] Test a todo with a simple output

    > This is a simple output

  - [ ] Test a todo with a complex output

    > This is a complex output

    ---

    ```md
    This is **text**
    ```

  - [ ] Test a todo with an output and subtasks

    > This is a complex output
    > With multiple lines
    > And even a blank line in the middle.

    - This
    - is
    - a
      - multi-depth
    - list

    - [ ] And this is a subtask after the output
    - [ ] Decide if we should add a blank like to the end

  - [~] Test Second Level Todo

    - [x] Test Third Level Todo 1

      - [ ] And a fourth

    - [x] Test Third Level Todo with a [user input]
    - [x] Test Third Level Todo 3
    - [x] Test Third Level Todo 4

  - [ ] And another second level one

- [ ] And now back to the top

## Contributors

- **ðŸ¤– DoBot:** <https://github.com/ronan/dobot>
- **Ronan Dowling:** <ronan@dowlings.me>
